src/main/webapp/app/quiz/participate/quiz.module.ts

```diff
@@ -25,6 +26,9 @@ const ENTITY_STATES = [...quizRoute];
         QuizComponent,
         MultipleChoiceQuestionComponent,
         DragAndDropQuestionComponent,
+        DragItemComponent,
```

Comment:
now you have some components twice. Please make sure to have all components only once